### PR TITLE
Add support for Epson network scanners

### PR DESCRIPTION
**Pull request by booxter**

### CODE REVIEW

**Comment by booxter** [comment]
I've tested it based on release-19.03. I don't have a setup with master to validate cherry-pick works on master. Is there a procedure to validate a patch against master except by upgrading to master? For a separate standing packages it's easy to install it with nix-env but this package involves configuration options and I am not yet sure how to build with options included but isolated from the host.

**Comment by risicle** [comment]
Clone another copy of the `nixpkgs` repo somewhere and do `nix-build . -A some.package.attrpath` from within it?

**Comment by booxter** [comment]
@risicle I can definitely build the package but it's hard to tell if it definitely works with the whole setup (sane config files etc.) unless I build the whole system using master and the PR and try it out.

**Comment by risicle** [comment]
You can take a simple (or complex) machine configuration and build a vm from it https://nixos.wiki/wiki/Cheatsheet#Reuse_a_package_as_a_build_environment which you can just launch with a single command.

**Comment by booxter** [comment]
@risicle these build tools are amazing! I've just build a VM with my configuration and this branch (that is based on master) and validated that `scanimage -L` shows the remote scanner and that `scanimage > file` generates an image.

**Comment by symphorien** [comment]
Looks good to me! 
I rebuilt my system with these commits rebased on 19.03. My (usb) scanner still works.

**Comment by booxter** [comment]
@symphorien thanks for checking. How do we get it merged? I believe you are marked as the package maintainer so I hoped you can nudge it in.

**Comment by symphorien** [comment]
Sorry, I don't have merge rights. We can only wait for someone who can to notice this PR.

File: pkgs/misc/drivers/epkowa/default.nix [hunk 1]
```diff
@@ -161,6 +161,33 @@ let plugins = {
 
     meta = common_meta // { description = "iscan esci s80 plugin for "+passthru.hw; };
   };
+  network = stdenv.mkDerivation rec {
+    pname = "iscan-nt-bundle";
+    version = "1.0.0";
+    ntPluginVersion = "1.1.1-1";
+
+    buildInputs = [ stdenv.cc.cc.lib ];
+    nativeBuildInputs = [ autoPatchelfHook ];
```
Comment (veprbl) [inline pkgs/misc/drivers/epkowa/default.nix]: If this works then, perhaps, other plugins could use this too.

**Review by veprbl — APPROVED** [review]

**Pull request merged**